config/http_config_test.go

```diff
@@ -507,46 +507,48 @@ func TestNewClientFromConfig(t *testing.T) {
 	}
 
 	for _, validConfig := range newClientValidConfig {
-		testServer, err := newTestServer(validConfig.handler)
-		if err != nil {
-			t.Fatal(err.Error())
-		}
-		defer testServer.Close()
+		t.Run("", func(t *testing.T) {
```

Comment:
Suggestion: How does this work when multiple tests have the same name? Can you add a description here? The index in the for loop would be good enough here.

Reply:
Empty name generates index (autocount), so I think this should be fine!